### PR TITLE
chore(dtfs2-none): fix typo

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
## Introduction :pencil2:
Generic copy updates and refactoring that I found as part of my poking around the service post rampup

## Resolution :heavy_check_mark:
Makes the example 8 digit companies house actually be 8 digits, and use constants where possible in tests

## Miscellaneous :heavy_plus_sign:
Adding spaces, punctuation, etc
Make constant naming consistent throughout codebase (e.g. INVALID_WITH_SPACE used everywhere, rather than also having INVALID_SPACE in some places)

### CODE REVIEW

**Review by AlexBramhill — APPROVED** [review]
could you update assignee (to yourself) and label (to type chore), then lgtm (I did your other pr so you can see what it looks like)

File: e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-supply-contract-validation.spec.js [hunk 1]
```diff
@@ -291,7 +291,7 @@ context('about-supply-contract', () => {
     // indemnifier companies house submit - providing a value which has a space
     //--------------------------------------------------------------------------
 
-    contractAboutSupplier.indemnifierCompaniesHouseRegistrationNumber().clear().type('8989898 ');
+    contractAboutSupplier.indemnifierCompaniesHouseRegistrationNumber().clear().type('89898989 ');
```
Comment (abhi-markan) [inline e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-supply-contract-validation.spec.js:294]: Constant please

Comment (abhi-markan) [reply e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-supply-contract-validation.spec.js:294]: Can all the instances be from constant defined in 'e2e-tests'

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: e2e-tests/gef/cypress/e2e/exporter/exporter-correspondence-address.spec.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ context('Incomplete exporter section - application details page', () => {
     it('completes the exporter section', () => {
       cy.visit(url);
       applicationDetails.exporterDetailsLink().click();
-      companiesHouse.regNumberField().type('8989898');
+      companiesHouse.regNumberField().type('89898989');
```
[the remark this answers is not in the document]
Comment (natdeanlewissoftwire) [reply e2e-tests/gef/cypress/e2e/exporter/exporter-correspondence-address.spec.js:38]: Yep, good idea :)

**Review by abhi-markan — DISMISSED** [review]

**Comment by abhi-markan** [comment]
@natdeanlewissoftwire failing E2E test.

**Comment by natdeanlewissoftwire** [comment]
> @natdeanlewissoftwire failing E2E test.

@abhi-markan fixed thanks - should be ready for re-approval

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**